### DISCUSSION TITLE
Whether date & time stored will be retrived in local timezone automatically by SDK?

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [pocketbase/pocketbase#2123] [Q&A]
In Dart sdk, I am converting DateTime value to iso string during storage as in sample code below. 
```
DateTime now=DateTime.now();
Map<String, dynamic> payload={
"date":now.toIso8601String()
}
```
During listing of records I am receiving date field as string, whether return value will be of UTC timezone or in local timezone? If we are developing app for different timezone users, how it will should be handled?

**Comment by ganigeorgiev** [comment]
The stored and returned datetime is always in UTC.
